modules/ossm-config-sampling.adoc

```diff
@@ -5,16 +5,16 @@
 [id="ossm-config-sampling_{context}"]
 = Adjusting the sampling rate
 
-The distributed tracing sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
+The Envoy sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
```

Comment:
The second sentence reads well, so I don't want to change it, but I do try to avoid unparalleled verbs, like going from active (consumes) to be verbs (is). 

Do you think this revision works, `A high sampling rate consumes cluster resources and performance but helps you with debugging issues.` ? 

Again, the original reads very well, so feel free to keep it as is :)